pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -78,6 +79,12 @@ stdenv.mkDerivation rec {
       # e.g., those from the "Effects" menu.
       python3 = "${python3Env}/bin/python";
     })
+    # Use the patch from the Arch Linux package repository to support
+    # Poppler 22.09.0.
+    (fetchpatch {
+      url = "https://raw.githubusercontent.com/archlinux/svntogit-packages/b2f65dfb60ae0c8cd6cd9affd3d71064082a6201/trunk/inkscape-1.2.1-poppler-22.09.0.patch";
+      sha256 = "sha256-pArvsS/qoCTMAisF8yj3agZKrb90zRFZkck1TX0KeGc=";
```

Comment:
```suggestion
      hash = "sha256-pArvsS/qoCTMAisF8yj3agZKrb90zRFZkck1TX0KeGc=";
```

Reply:
`https://github.com/archlinux/svntogit-packages/raw/b2f65dfb60ae0c8cd6cd9affd3d71064082a6201/trunk/inkscape-1.2.1-poppler-22.09.0.patch` makes it easier to go back to the git repo.